src/main/java/Pages/DiffElementsPage.java

```diff
@@ -0,0 +1,46 @@
+package Pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class DiffElementsPage {
```

Comment:
All radioButtons and checkboxes should be clicked by name from the list of such elements. Don't specify each of them separately.